pages/common/git-merge-base.md

```diff
@@ -0,0 +1,16 @@
+# git merge-base
+
+> Find a common ancestor of two commits.
+> More information: <https://git-scm.com/docs/git-merge-base>.
+
+- Output a best common ancestor of two commits:
+
+`git merge-base {{commit_1}} {{commit_2}}`
```

Comment:
```suggestion
`git merge-base {{commit1}} {{commit2}}`
```

Reply:
The page for git-range-diff also uses underscores, so I added these for consistency. Should I remove them here and also change the other page (in a separate PR maybe)?

Reply:
Leave as-is.

Reply:
There is no rule, but I think we don't use `_` in most cases.

Reply:
> The page for git-range-diff also uses underscores, so I added these for consistency. Should I remove them here and also change the other page (in a separate PR maybe)?

All pages should be consistent, and not just some of them in this case. Numbering it's what can be automatically fixed via regex substitutions via Bash script.